posthog/celery.py

```diff
@@ -82,7 +82,7 @@ def receiver_setup_logging(loglevel, logfile, format, colorize, **kwargs) -> Non
 
     # following instructions from here https://django-structlog.readthedocs.io/en/latest/celery.html
     # mypy thinks that there is no `logging.config` but there is ¯\_(ツ)_/¯
-    logging.config.dictConfig(logs.LOGGING)  # type: ignore
+    logging.config.dictConfig(logs.LOGGING)
```

Comment:
This is marked as an `unused-ignore` by mypy, but I couldn't get `mypy-baseline` sync to understand it was not fixed. So I've "fixed" it.

Reply:
Nevermind, this pops up as a different error now. I've reverted this "fix".